src/components/CollectionFilters/collectionFilters.scss

```diff
@@ -123,3 +123,34 @@
     }
   }
 }
+
```

Comment:
it feels like some of these values might want to be variables. Especially if they relate to one another -- like if two should be equal to each other.